app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -44,13 +46,15 @@ function ServiceInstanceRow($filter, DataService, rowMethods, $uibModal) {
     row.notifications = rowMethods.getNotifications(row.apiObject, row.state);
     row.displayName = getDisplayName();
     row.description = getDescription();
-    row.instanceBindings = getBindings();
   };
 
   row.getSecretForBinding = function(binding) {
     return binding && _.get(row, ['state', 'secrets', binding.spec.secretName]);
   };
 
+  row.showSecretLink = function(binding) {
+    return statusConditionReady(binding);
```

Comment:
I think it's more clear if you just call the filter directly in the view. I was confused why `!showSecretLink()` means show a pending icon, whereas

```html
          <span ng-if="!(binding | statusConditionReady)">
             <status-icon status="'Pending'"></status-icon> Pending
          </span>
```

is pretty clear